docs/api/QUIC_STREAM_EVENT.md

```diff
@@ -156,7 +163,7 @@ Client context to match this event with the original `StreamSend` operation.
 
 ## QUIC_STREAM_EVENT_PEER_SEND_SHUTDOWN
 
-Indicates that the current send operation to the peer on this stream has been shutdown/completed.
+Indicates that the current send operation from the peer on this stream has been completed/shutdown.
```

Comment:
I don't think we should use "send operation" here: in MsQuic, we use "send operation" to mean a call to StreamSend.
Maybe we can use "the send direction of the stream" instead?

I would also say "gracefully completed/shutdown" just to be very clear this isn't emmited on an abortive shutdown.

Reply:
Ditto.